server/accounts.go

```diff
@@ -793,6 +842,20 @@ func (a *Account) checkExpiration(claims *jwt.ClaimsData) {
 	a.expired = false
 }
 
+func (a *Account) hasValidIssuer(juc *jwt.UserClaims) bool {
+	a.mu.Lock()
```

Comment:
Should this be the read lock instead? (I don't see that we set anything in that function, just read)

Reply:
Fixed